src/hooks/comments/service.ts

```diff
@@ -0,0 +1,13 @@
+import { axiosClient } from '@/apis';
+import { formatOptionQuery, IFormatOptionQuery } from '@/utils/getPagiantion';
+import { format } from 'path';
+import { PaginationServicer } from '../interfaces';
+import { ResponseGetComments } from './interface';
+
+class CommentService implements PaginationServicer<ResponseGetComments> {
+  basePublicURL: string = '/public/comments';
```

Comment:
Should be `baseUrl` or `commentServiceBaseUrl`.

Reply:
in my opinion , in here CommentService have 2 url is : public and private , it't have only baseUrl